frontend_lib/src/component/HTMLContent/HTMLContent.jsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { CUSTOM_EVENT } from '../../customEvent.js'
-// require('./HTMLContent.styl')
+
+import Prism from 'prismjs'
+
+import 'prismjs/plugins/line-numbers/prism-line-numbers.js'
+import 'prismjs/plugins/toolbar/prism-toolbar.js'
+import 'prismjs/plugins/copy-to-clipboard/prism-copy-to-clipboard.js'
+
+import 'prismjs/components/prism-java'
+
+/*
+  INFO - G.B. - 2022-09-28 - Apparently to add a plugin, you just import it, the rest is done automatically by the Prism.highlightAll() command.
+  The same is true for the support of new languages.
+  Also, a new css file has been added in frontend/dist/assets/prism.css.
```

Comment:
I'd like you to put the prism.css file in a prism folder instead of at the root of assets/